src/main/java/org/gridsuite/study/server/repository/rootnetwork/RootNetworkNodeInfoRepository.java

```diff
@@ -42,6 +42,8 @@ public interface RootNetworkNodeInfoRepository extends JpaRepository<RootNetwork
 
     List<RootNetworkNodeInfoEntity> findAllByPccMinResultUuidNotNull();
 
+    List<RootNetworkNodeInfoEntity> findAllByNodeExportNetworkExportUuid(UUID exportUuid);
```

Comment:
I'm not sure this method is relevant. Rather add a JPQL request (as it is done for existsByNodeUuidsAndBlockedNode()) that will return a NodeExportEmbeddable for a given  exportUuid. This will simplify also updateExportNetworkStatus(UUID exportUuid, ExportNetworkStatus status)